signing/examples/BUILD

```diff
@@ -17,7 +17,9 @@ offline_presigning_artifacts(
     # key (e.g. //sw/device/silicon_creator/rom/keys/real/rsa:earlgrey_a0_test_0).
     # This is left as a fake key so that the presigning artifacts will be
     # appropriate for the `fake_sign` rule below.
-    rsa_key = "//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0",
+    rsa_key = {
+        "//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "fake_test_key_0",
```

Comment:
qq to make sure I understand: 

1. Is the value only used to generate the produced filename suffix? 
2. Should we add an example to drive sining with a real test key? Assuming the target just changes here to point to a public key.

Reply:
Currently, this value is used to derive the filename suffix.  When I add support for signing with a token, the label will be the keyset (which will contain the public key information) and the value will be the PKCS#11 label of the signing key.

I'm not completely satisfied that this field can have different inputs depending on the key type (local fake private key vs. on-token keys), but this allows us to maintain compatibility with the existing signing methodology test programs.

Reply:
ack. I missed the dual use aspect of the dict value. We should capture the conventions in a follow up PR as part of the signing documentation.